### PR TITLE
Remove Function CR refs

### PR DESCRIPTION
**Pull request by efiacor**
The Function CRD and Repo content tyoe has been removed from Porch. This cleans up any remaining refs.

### CODE REVIEW

**Review by kushnaidu — COMMENTED** [review]
Small typo in the PR title else everything looks good to me.

File: content/en/docs/porch/contributors-guide/_index.md [hunk 1]
```diff
@@ -113,4 +113,4 @@ then run test suite against the Porch instance.
 
 [VSCode](https://code.visualstudio.com/) works really well for editing and debugging.
```
Comment (CsatariGergely) [inline content/en/docs/porch/contributors-guide/_index.md:114]:
```suggestion
[VS Code](https://code.visualstudio.com/) works really well for editing and debugging.
```

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

**Comment by CsatariGergely** [comment]
We can ignore the reviewdog job result so far. It is fixed in https://github.com/nephio-project/docs/pull/176 what requires the merger of https://github.com/nephio-project/docs/pull/179

**Review by liamfallon — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**